backend/src/main/java/org/cryptomator/hub/filters/VaultAdminOnlyFilterProvider.java

```diff
@@ -48,11 +52,9 @@ public void filter(ContainerRequestContext containerRequestContext) {
 	void verify(JWTVerifier verifier, DecodedJWT vaultAdminAuthorizationJWT) {
 		try {
 			verifier.verify(vaultAdminAuthorizationJWT);
-		} catch (TokenExpiredException e) {
-			throw new VaultAdminTokenExpiredException("Token of VaultAdminAuthorizationJWT expired");
 		} catch (IncorrectClaimException e) {
-			if (e.getClaimName().equals(RegisteredClaims.ISSUED_AT) || e.getClaimName().equals(RegisteredClaims.NOT_BEFORE)) {
-				throw new VaultAdminTokenNotYetValidException("Token of VaultAdminAuthorizationJWT not yet valid");
+			if (e.getClaimName().equals(RegisteredClaims.ISSUED_AT)) {
+				throw new VaultAdminValidationFailedException("Token of VaultAdminAuthorizationJWT not provided or not yet or no longer valid");
```

Comment:
I really like the old exception better. We specifically test if the IAT claim is the culprit, so why should it be "VaultAdminAuthorizationJWT not provided"?

Reply:
The message wasn't correct, now it fits better: https://github.com/cryptomator/hub/pull/164/commits/42e69c0ddc2006e41a41080a28bfd94e7029d58c

Sure, if we want to throw a more specific exception, we need to check the cause again here... :thinking: